packages/contracts-bridge/contracts/BridgeRouter.sol

```diff
@@ -36,11 +37,13 @@ contract BridgeRouter is Version0, Router {
     ITokenRegistry public tokenRegistry;
     // token transfer prefill ID => LP that pre-filled message to provide fast liquidity
     mapping(bytes32 => address) public liquidityProvider;
+    // reference to connext contract on this domain
+    IConnext public connext;
```

Comment:
**Note:** would _prefer_ not to store this in the router. would be ideal to have a more generalized feature - the caller passes in the token recipient + call recipient + call data 🤔

Reply:
agree. this should be a user-specified message field